### ISSUE TITLE
Save Python wheel packages in a separate folder

### ISSUE THREAD

**Issue by daniandtheweb** [lamikr/rocm_sdk_builder#95]
It could be nice to copy all the built Python wheel packages into a separate folder, this way it would be easier to install them in a python virtual environment without the need to use the --system-site-packages and it could also improve the testing process by having different package versions ready to test saved in there without the need for rebuilding every time.

**Comment by lamikr** [comment]
I agree, that it's good idea to save them somewhere. Any opinion for having /opt/rocm_sdk_xyz/install/python directory? (If there is some interest for creating rpm/deb files later from the installed files, those could the put under install dir also)

Btw, do you know that if you only want do the install part of the build, it can be done by removing all
.result_install and .result_postinstall files under builddir and then launching the build command again. If the rest of the .result_xyz files are in place, it will skip those steps and just do the install and postinstall steps.

```
cd builddir
find -name .result_install | xargs rm -f
find -name .result_postinstall | xargs rm -f
cd ..
./babs.sh -b
```

**Comment by daniandtheweb** [comment]
Yes, I already know about that, what I meant is that by having the wheels saved somewhere else they're saved from being deleted during a clean build, allowing to be able to test different versions easily.

I think that the best way could be to just save the wheels in a folder like rocm_sdk_builder/python_wheels/. Using the opt folder for files that have to be easily reachable in order to be installed in a virtual environment seems a little odd to me.

**Comment by lamikr** [comment]
So we have 3 options
1)  totally separate folder for example in user root dir 
2) Somewhere under rocm_sdk_builder build source directory (I think you suggested this)
3) I suggeted that somewhere under rocm sdk install target directory (my suggestion)

- Weak point of option 1 is that is that the number of wheel filen can grow huge without user realizing it and files there does not have neccessarily have any direct association with the build
- Weak point of option 2 is that sometimes user may want to update the build directory and again the wheel files does not neccessary reflect anymore with the source code/release number. Or even swipe out the source directory after build and just copy the install directory under opt from virtual machine to live environment with gpu's available
- Weak point of option 3 is that if user wants to delete the install directory, then also the wheel files would get deleted.

I would still favor either option 2 or 3 and I think it's possible to find use case favoring both of them.

**Comment by lamikr** [comment]
I started working this and also noticed that I had broken the whl file generation by compining the build and install scripts to one and just always calling "setup.py install" without bw_wheel. I for some reason thought that all packages will create the wheel when calling the install and then install that one. Instead it causes the creation of egg files that will be deprecated soon.
